### DISCUSSION TITLE
API down?

### DISCUSSION THREAD

**Discussion by nschloe** [keygen-sh/community#122] [Issue]
```
ping api.keygen.sh
```
```
PING behavioural-mandrill-rnma91gjivqr3isoab4hacmq.herokudns.com (3.220.57.224) 56(84) bytes of data.
```

https://app.keygen.sh/ seems down as well.

_Edit:_ I just saw https://status.keygen.sh/, so yeah.

_Edit:_ Now I'm getting API responses, but partly with incorrect signatures. -- This can probably be called a bug.

**Comment by asaf-b** [comment]
Same issues here.
ETA on getting the API back up?

**Comment by ezekg** [comment]
We experienced a DDoS this morning. During mitigation, some parts of our API sporadically failed, such as signature verification (due to Cloudflare's CDN modifying the `Date` header, causing it to become out of sync with the `Date` header that was signed).

We're writing a post-mortem that will detail this in further depth once we're able to fully review the incident.

**Reply by nschloe** [reply]
For reference: https://status.keygen.sh/incident/322421
